docs/examples/SPA.md

```diff
@@ -0,0 +1,66 @@
+## Using offline-plugin in a Single Page Application (SPA)
+
+In a SPA, only the inital HTML is served from the server. Any routing beyond that takes place on the client. So how should we configure offline-plugin to handle this?
```

Comment:
@NekR fair enough. I was thinking in terms of a server-side rendered application, where the route is resolved and the correct html is generated on the server. Will try to fomulate it without that assumption.